consts/categories.js

```diff
@@ -1,19 +1,64 @@
 const categories = {
-  LANGUAGES: 'languages',
-  MATHEMATICS: 'mathematics',
-  COMPUTER_SCIENCE: 'computer science',
-  MUSIC: 'music',
-  DESIGN: 'design',
-  HISTORY: 'history',
-  BIOLOGY: 'biology',
-  PAINTING: 'painting',
-  FINANCES: 'finances',
-  MARKETING: 'marketing',
-  AUDIT: 'audit',
-  CHEMISTRY: 'chemistry',
-  ASTRONOMY: 'astronomy',
-  PHYSICS: 'physics',
-  PSYCHOLOGY: 'psychology'
+  LANGUAGES: {
+    name: 'languages',
+    categoryIcon: 'mocked-path-to-icon'
```

Comment:
shouldn't it have pic format? i believe path must have .jpg or .png

Reply:
It will be changed soon and I decided to place a temporary string used just to indicate what will be there in future. So, is it necessary to specify format?

Reply:
i believe it makes sense to do now, coz i have doubts that it will be changed